README.md

```diff
@@ -95,6 +95,7 @@ Code written in Move.
 - [CoinSwap](https://github.com/move-language/move/tree/main/language/documentation/examples/experimental/coin-swap) - A toy implementation of a [Uniswap](https://uniswap.org/)-like liquidity pool containing two tokens.
 - [StarSwap](https://github.com/Elements-Studio/starswap-core) - A Uniswap-style DEX. Deployed on Starcoin.
 - [Offer](https://github.com/move-language/move/blob/main/language/move-stdlib/nursery/sources/offer.move) - Generic implementation of atomic swaps for any pair of assets.
+- [OmniBTC](https://www.omnibtc.finance/) - A omnichain financial platform for web3, including omnichain swap and BTC omnichain lending.
```

Comment:
Could you please link to Move code for OmniBTC rather than to a product page here (as the other links do)?

The intention of this list is to provide useful code examples that other Move developers can learn from.

Reply:
https://github.com/OmniBTC/OmniSwap, This is the omnichain Swap code we are using now, and it already supports public chains such as ETH/Layer2/BSC/avax. A version of the Move language is under development and will be open sourced soon.

Reply:
Great! Once the code is open-sourced, feel free to update this PR with a link. But we should not merge it until then.

Reply:
Thank you, we are working hard to realize the development

Reply:
OmniBTC = OmniSwap + OmniLending + OmniBridge.

The current open source project is just a single currency pool in OmniLending. The function provided is a simple function for users to withdraw coins from a single coin pool. Other modules, we are developing synchronously, will be open source one after another.

https://github.com/OmniBTC/MoveCoinPool